### PR TITLE
chore: differentiate cpu usage metrics and use avg_cpu_usage

### PR DESCRIPTION
**Pull request by Ziinc**
Adjusts the attribute to use `avg_cpu_usage`, which is clearer which cpu usage metric is in use. 
This change is backwards compatible and does not require a companion migration to be merged, as data is stored as is when saved.
Migration will only be needed when removing support for `cpu_usage` metric on platform  api.
Screenshot 2023-06-24 at 3 35 47 AM

### CODE REVIEW

**Comment by chasers** [comment]
The new max metric should be used here because that's what we're calling it and that's what people should be concerned with. 

Screenshot 2023-06-24 at 6 42 36 AM

File: studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const Infrastructure = ({
 
   const { data: cpuUsageData, isLoading: isLoadingCpuUsageData } = useInfraMonitoringQuery({
     projectRef,
-    attribute: 'cpu_usage',
+    attribute: 'avg_cpu_usage',
```
Comment (chasers) [inline studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx:63]:
```suggestion
    attribute: 'max_cpu_usage',
```

File: studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ const Infrastructure = ({
   )
 
   const chartMeta: { [key: string]: { data: DataPoint[]; isLoading: boolean } } = {
-    cpu_usage: {
+    avg_cpu_usage: {
```
Comment (chasers) [inline studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx:94]:
```suggestion
    max_cpu_usage: {
```

File: studio/components/interfaces/BillingV2/Usage/Usage.constants.tsx [hunk 3]
```diff
@@ -50,10 +50,10 @@ export const USAGE_CATEGORIES: CategoryMeta[] = [
       {
         anchor: 'cpu',
         key: 'cpu_usage',
-        attribute: 'cpu_usage',
+        attribute: 'avg_cpu_usage',
```
Comment (chasers) [inline studio/components/interfaces/BillingV2/Usage/Usage.constants.tsx:53]:
```suggestion
        attribute: 'max_cpu_usage',
```

File: studio/components/interfaces/BillingV2/Usage/Usage.constants.tsx [hunk 4]
```diff
@@ -50,10 +50,10 @@ export const USAGE_CATEGORIES: CategoryMeta[] = [
...
         name: 'CPU',
         unit: 'percentage',
-        description: 'CPU usage of your server',
+        description: 'Average CPU usage of your server',
```
Comment (chasers) [inline studio/components/interfaces/BillingV2/Usage/Usage.constants.tsx:56]:
```suggestion
        description: 'Max CPU usage of your server',
```

**Comment by Ziinc** [comment]
Gotcha, will let @joshenlim have a quick look at it before merging.

**Review by chasers — APPROVED** [review]

**Comment by kevcodez** [comment]
Screenshot 2023-06-27 at 01 35 18

Preview is crashing for me on the usage page

**Comment by Ziinc** [comment]
Screenshot 2023-06-28 at 2 57 26 AM

fix verified on staging preview.

**Comment by Ziinc** [comment]
added in a test for infra section rendering as well.

**Pull request merged**